src/utils_format_kairosdb.c

```diff
@@ -181,7 +181,9 @@ static int values_to_kairosdb(char *buffer, size_t buffer_size, /* {{{ */
 
 static int value_list_to_kairosdb(char *buffer, size_t buffer_size, /* {{{ */
                                   const data_set_t *ds, const value_list_t *vl,
-                                  int store_rates) {
+                                  int store_rates,
+                                  char const *const *http_attrs,
+                                  const int http_attrs_num) {
```

Comment:
I reviewed this before, but that probably got lost. `const int` makes no sense in function parameters, please make this just an int. Also mentioned before, you declare this in write_http.c as a size_t, so pick one or the other.